component/receiver.go

```diff
@@ -171,43 +156,46 @@ type receiverFactory struct {
 	logsStabilityLevel StabilityLevel
 }
 
+// Deprecated: [v0.67.0] use receiver.TracesStabiilty.
 func (r receiverFactory) TracesReceiverStability() StabilityLevel {
 	return r.tracesStabilityLevel
 }
 
+// Deprecated: [v0.67.0] use receiver.MetricsStabiilty.
 func (r receiverFactory) MetricsReceiverStability() StabilityLevel {
 	return r.metricsStabilityLevel
 }
 
+// Deprecated: [v0.67.0] use receiver.LogsStabiilty.
```

Comment:
I don't think we deprecated this correct? We should, but that is a change in the Factory interface and will do later.

Reply:
Yeah, good catch.